pkg/qemu-ga/Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: tag push
+
+ORG?=linuxkit
+IMAGE=qemu_ga
```

Comment:
OTOH we have `pkg/open-vm-tools` which doesn't have this problem and is much the same from a packaging perspective. I'm going to try switching it all back to `qemu-ga` and see what breaks.